### PR TITLE
Feature branch: index and about.html with functionalities 

### PR DESCRIPTION
**Pull request by rcqx**
# Report of updates to the main branch

* Index and about HTML files were created
* Open BIM conference 2022 was the main topic of this website
* Both mobile and desktop versions of the website were implemented
* Vanilla CSS was mostly used for styling the page with some exceptions in main containers (using container and container-fluid classes from bootstrap)
* Dynamic creation of HTML elements was implemented using JavaScript (Speakers cards)
* Mobil menu was also developed in HTML and CSS and functionalities were added with JS
* Other user interactions such as links and buttons were added to the project
* The project was deployed using GitHub pages service 
* README file was added
* Brief video describing the project was added according to capstone curriculum guidelines

[LOOM VIDEO HERE](https://www.loom.com/share/19727339f9944b24a12f650eb288bd52)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
-# capstone-project-1
-HTML/CSS &amp; JavaScript capstone project
+# HTML/CSS & JavaScript capstone project - Conference page 🏅!
+
+Online website for the open BIM global conference. This project followed the design guidelines from 
+
+![CC Global Summit 2015 design by Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015])
```
Comment (elmar8287) [inline README.md]: - [ ] Please, try to fix this broken image
Screen Shot 2022-04-15 at 00 03 21

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+
+Give a :star: if you like this project!
```
Comment (elmar8287) [inline README.md:78]: - [ ] Please, consider that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        <img src="static/ifclogo.png" alt="facebook logo">
+        <h3>Open BIM Global <br> Summit 2022</h3>
+      </div>
+      <ul class="conta">
+        <li><a href="#about">About</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="https://www.buildingsmart.org/about/openbim/">Join</a></li>
+        <li><a href="index.html#sponsors">Sponsor</a></li>
+        <li><a href="https://www.buildingsmart.org/resources/latest-news/">News</a></li>
+        <li><a id="aButton" href="https://www.buildingsmart.org/chapter-directory/">BIM Campaing</a></li>
```
Comment (elmar8287) [inline about.html:42]: - [ ] Please, consider that there is no possibility to navigate to the Home page from the About page, both desktop and mobile versions.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>About openBIM</title>
+</head>
+
+<body>
+  <div id="navWrapper">
+    <nav class="topnav">
+      <div id="navLinksWrapper">
+        <img src="static/facebook.png" alt="">
+        <img src="static/twitter.png" alt="twitter logo">
```
Comment (elmar8287) [inline about.html]: - [ ] Please, consider on whole pages (both Home and About) adding alternative text for images. You did it for some images 👍 but there still needs improvement 👍

File: about.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    </p>
+    <p id="logoTextDesktop">The logo of Open BIM Global Developers Summit 2022 was decided throught the Logo
+      competition from october 8th to november 8th. After online public voting and Selection Committee's judging
+      process,
+      the logo from Juan Perez won the prize. After consultation with Mr. Agrawal we optimized the prize-winning-logo
+      for
+      this year's summit as following.
+    </p>
+    <div class="logo">
+      <img src="static/openbim.jpg" alt="">
```
Comment (elmar8287) [inline about.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

File: about.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    <div class="sponsors">
+      <h5>IFCjs</h5>
+      <h5>UCL</h5>
+      <h5>UAM</h5>
+      <h5>Arup</h5>
+      <h5>BSmart</h5>
+    </div>
+  </section>
+  <footer>
+    <img src="static/ifclogo.png" alt="">
```
Comment (elmar8287) [inline about.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

File: index.html [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>openBIM Summit 2022</title>
+</head>
+
+<body>
+  <div id="navWrapper">
+    <nav class="topnav">
+      <div id="navLinksWrapper">
+        <img src="static/facebook.png" alt="">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

File: index.html [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+      <h2 class="coco">2022.12.02 (FRI) ~ 03 (SAT)</h2>
+      <p>@Camino Real Polanco Conference Center CDMX</p>
+    </div>
+  </section>
+  <section id="program" class="container-fluid">
+    <h2>Main Program</h2>
+    <div class="break"></div>
+    <div class="events">
+      <div class="event">
+        <img src="static/program1.png" alt="">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

File: index.html [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+        <h5>Lecture</h5>
+        <p>Listen to the speakers from various contries about the messages of open BIM</p>
+      </div>
+      <div class="event">
+        <img src="static/program2.png" alt="">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

File: index.html [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+        <h5>Forum</h5>
+        <p>Have the time to share your thoughts and opinions with open BIM and AECO experts</p>
+      </div>
+      <div class="event">
+        <img src="static/program4.png" alt="">
+        <h5>Network</h5>
+        <p>Try creating your own open BIM tool using available resources such as IFCjs</p>
+      </div>
+      <div class="event">
+        <img src="static/program5.png" alt="">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

File: index.html [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+    <div class="sponsors">
+      <h5>IFCjs</h5>
+      <h5>UCL</h5>
+      <h5>UAM</h5>
+      <h5>Arup</h5>
+      <h5>BSmart</h5>
+    </div>
+  </div>
+  <footer>
+    <img src="static/ifclogo.png" alt="">
```
Comment (elmar8287) [inline index.html]: - [ ] Please, try to fix alt issues, as I mentioned before 🥇

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @jr-cast.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues 👍 
- Nice design! 💯 👍 

## Required Changes ♻️

- [ ] Regarding the video: Nice presentation 👍 but as required try to add highlighted piece of the code you find interesting or are proud of making. 
- [ ] Check the comments under the review.

## Optional suggestions

- Please, try to use in the future the commits titles with a capital letter. You have a good commits history, but all with small letters - make it more professional 🥇 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @jr-cast, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Hamzaoutdoors) [reply README.md]: ✔️ well resolved

File: README.md (hunk 2 above)
Comment (Hamzaoutdoors) [reply README.md:78]: ✔️ well resolved

File: about.html (hunk 3 above)
Comment (Hamzaoutdoors) [reply about.html:42]: ✔️ well resolved

File: about.html (hunk 4 above)
Comment (Hamzaoutdoors) [reply about.html]: ✔️ well resolved

File: about.html (hunk 5 above)
Comment (Hamzaoutdoors) [reply about.html]: ✔️ well resolved

**Pull request merged**